.circleci/unittest/windows/scripts/set_cuda_envs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+if [ "${CU_VERSION:-}" != "cpu" ] ; then
+    if [[ ${#CU_VERSION} -eq 5 ]]; then
+        CUDA_VERSION="${CU_VERSION:2:2}.${CU_VERSION:4:1}"
+    fi
+
+    # It's a log to see if CU_VERSION exists, if not, we use environment CUDA_VERSION directly
+    echo "Using CUDA $CUDA_VERSION as determined by CU_VERSION $CU_VERSION"
+
+    version="$(python -c "print('.'.join(\"${CUDA_VERSION}\".split('.')[:2]))")"
```

Comment:
I'm having a hard time understanding what this statement is doing. CUDA_VERSION is already MAJOR.MINOR, why we need to split and join it again?

Reply:
I remember I came across the passed CUDA_VERSION  is 11.2.2. (MAJOR.MINOR.UPDATE)
So I use [:2] to ensure the version number I use is MAJOR.MINOR

Reply:
To avoid confusion, I remove it. MAJOR.MINOR.UPDATE should be a incorrect input.